gapii/cc/to_proto.h

```diff
@@ -94,53 +72,17 @@ struct ProtoConverter<::google::protobuf::RepeatedField<Out>*, core::StaticArray
     }
 };
 
-template <typename EntryOut, typename KeyOut, typename KeyIn>
```

Comment:
Same as above. The only reason these are unused is that we have not up until now had a map in something that we serialize but with the push to serialize the state-block, many of these types are needed.

Reply:
I actually did not remove the map.  I just simplified the code.  Maps are currently used in GLES.